src/Fantomas.Tests/TupleTests.fs

```diff
@@ -268,3 +268,61 @@ let y =
          sprintf "(%s)" args),
     namesWithIndices
 """
+
+[<Test>]
+let ``comment on first tuple argument is preserved`` () =
+    formatSourceString
+        false
+        """
+let func (a, b) = a + b
+
+func(
+        // abc
+        0,
+        1
+)
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+let func (a, b) = a + b
+
+func (
+    // abc
+    0,
+    1
+)
+"""
+
+[<Test>]
+let ``comment trivias on tuple arguments are preserved`` () =
+    formatSourceString
+        false
+        """
+let func (a, b) = a + b
+
+func(
+        // abc
+        0, // def
+        // ghi
+        1 // jkl
+        // mno
+)
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+let func (a, b) = a + b
+
+func (
+    // abc
+    0,  // def
+    // ghi
+    1 // jkl
+// mno
```

Comment:
Well, this is another bug related to the assignment of trivia to a trivia node.
In short, Fantomas detected the comment `// mno` and has to find an anchor to attach it.
It chose to add it before the closing parenthesis `)` instead of after the `SynExpr.Tuple`.
I'm aware of these situations, but it is quite complex to solve to be fair.
For this PR you can leave it like that.